### PR TITLE
Update teams-live-event-usage-report.md

### PR DESCRIPTION
**Pull request by chbalaki**
Updating documentation to reflect existing behavior.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-analytics-and-reports/teams-live-event-usage-report.md [hunk 1]
```diff
@@ -42,7 +42,9 @@ The Teams live event usage report in the Microsoft Teams admin center shows you
 |**5**   |Select **Edit columns** to add or remove columns in the table.|
 
 ## Notes
-Showing up to 100 live events that match the current report criteria. To see more live events, apply date filters to reduce the list size.
+We show up to 100 live events that match the current report criteria. To see more live events, apply date filters to reduce the list size. 
+
+Users who watch the recording of the event or the event on demand will not be included in the view count.
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/teams-live-event-usage-report.md:47]:
```suggestion
Everybody who watches the recording of the event or the event on demand will not be included in the view count. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @chbalaki, kindly commit the minor suggested change if you agree with it. As per the MS Style Guide, we avoid using the word "users" where possible. Many thanks.

**Comment by chbalaki** [comment]
Looks good to me! I have committed that suggestions: Thank you!

**Comment by chbalaki** [comment]
Also @JohanFreelancer9 - how do I fully commit the PR and merge it with the Main branch?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by chbalaki** [comment]
@scanum, @SerdarSoysal - can you please help me merge these requests? This PR has been open for a long time and we have a customer escalation around this.

**Pull request merged**